packages/data-broker/src/__tests__/DataBroker/DataBroker.test.js

```diff
@@ -3,73 +3,52 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-import { expect } from 'chai';
-import sinon from 'sinon';
-
 import { DataBroker } from '../../DataBroker';
 import * as GetModels from '../../getModels';
 import { DATA_BY_SERVICE, DATA_ELEMENTS, DATA_GROUPS } from './DataBroker.fixtures';
 import { stubCreateService, createModelsStub, createServiceStub } from './DataBroker.stubs';
 
 describe('DataBroker', () => {
-  let createService;
   const dataSources = Object.values(DATA_ELEMENTS).concat(Object.values(DATA_GROUPS));
   const models = createModelsStub(dataSources);
   const SERVICES = {
     test: createServiceStub(DATA_BY_SERVICE.test),
     other: createServiceStub(DATA_BY_SERVICE.other),
   };
-
+  const createServiceMock = stubCreateService(SERVICES);
   const options = { ignoreErrors: true, organisationUnitCode: 'TO' };
 
-  before(() => {
-    createService = stubCreateService(SERVICES);
-    sinon.stub(GetModels, 'getModels').returns(models);
-  });
-
-  beforeEach(() => {
-    createService.resetHistory();
-    Object.values(SERVICES).forEach(service => {
-      service.push.resetHistory();
-      service.delete.resetHistory();
-      service.pull.resetHistory();
-      service.pullMetadata.resetHistory();
-    });
-  });
-
-  after(() => {
-    createService.restore();
-    GetModels.getModels.restore();
+  beforeAll(() => {
+    jest.spyOn(GetModels, 'getModels').mockReturnValue(models);
   });
 
   it('getDataSourceTypes()', () => {
-    expect(new DataBroker().getDataSourceTypes()).to.deep.equal(models.dataSource.getTypes());
+    expect(new DataBroker().getDataSourceTypes()).toStrictEqual(models.dataSource.getTypes());
   });
 
   describe('push()', () => {
     it('throws if the data sources belong to multiple services', async () => {
       const data = [{ value: 1 }, { value: 2 }];
       const dataBroker = new DataBroker();
       return expect(
-        dataBroker.push({ code: ['TEST_01', 'OTHER_01'], type: 'dataElement' }),
-        data,
-      ).to.be.rejectedWith('Cannot push data belonging to different services');
+        dataBroker.push({ code: ['TEST_01', 'OTHER_01'], type: 'dataElement' }, data),
+      ).toBeRejectedWith('Cannot push data belonging to different services');
     });
```

Comment:
I think the `data` should be one of the params in `.push` instead of `expect`

Reply:
Oh yeah great catch!! 👀